pkg/registrationserver/registrationserver.go

```diff
@@ -0,0 +1,126 @@
+package registrationserver
+
+import (
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/codeready-toolchain/registration-service/pkg/configuration"
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+	errs "github.com/pkg/errors"
+)
+
+var (
+	// Commit current build commit set by build script
```

Comment:
this could be moved in the `configuration` pkg

Reply:
Agree, those should be moved to the configuration package (though not the configuration options).